content/api/commands/intercept.md

```diff
@@ -170,116 +94,153 @@ The `routeHandler` defines what will happen with a request if the [routeMatcher]
 
-// intercepts _limit=5 requests
-cy.get('#load-five-users').click()
-cy.wait('@users').its('response.body').should('have.length', 5)
-```
+```js
+cy.intercept('**/users')
+// matches this: GET http://localhost/users
+// ...and this, too: POST http://localhost/users
 
-### Waiting on a request
+cy.intercept('GET', '**/users')
+// matches this: GET http://localhost/users
+// ...but not this: POST http://localhost/users
+```
 
-Use [cy.wait()](/api/commands/wait) with `cy.intercept()` aliases to wait for the request/response cycle to complete.
+### Using routeMatcher
 
-#### With URL
+Specifying a `method` and `url` to match can also be acheived by passing the `routeMatcher` object into `cy.intercept` instead:
 
 ```js
-cy.intercept('http://example.com/settings').as('getSettings')
-// once a request to http://example.com/settings responds, this 'cy.wait' will resolve
-cy.wait('@getSettings')
+// These both yield the same result:
+cy.intercept({ method: 'GET', url: '**/users'})
+cy.intercept('GET', '**/users')
 ```
+`routeMatcher` is ideal when you require more [advanced matching]().
 
-#### With [RouteMatcher](#routeMatcher-RouteMatcher)
+### Aliasing a Route
+
+While `cy.intercept` doesn't yield anything, you can chain `as` off it to create an (`alias`)[]:
 
 ```js
-cy.intercept({
-  url: 'http://example.com/search*',
-  query: { q: 'expected terms' },
-}).as('search')
-
-// once any type of request to http://example.com/search with a querystring containing
-// 'q=expected+terms' responds, this 'cy.wait' will resolve
-cy.wait('@search')
+cy.intercept('GET', '**/users').as('getAllUsers')
+cy.intercept('POST', '**/users').as('createUser')
 ```
 
-#### Using the yielded object
+<!-- TODO move this -->
+`cy.intercept` can be used to passively listen for matching routes and apply aliases to them without manipulating the request or its response in any way. As you'll see, this alone is powerful as it allows you to wait for these requests, resulting in more reliable tests.
+
+### Waiting on a request
 
-Using [cy.wait()](/api/commands/wait) on a `cy.intercept()` route alias yields an interception object which represents the request/response cycle:
+We can build upon the previous examples of defining an aliased route with [cy.wait()](/api/commands/wait) in order to wait for the request to be sent and a response to be received before executing the next command.
 
 ```js
-cy.wait('@someRoute').then((interception) => {
-  // 'interception' is an object with properties 'id', 'request' and 'response'
-})
+cy.intercept('GET', '**/users').as('getAllUsers')
+cy.intercept('POST', '**/users').as('createUser')
+
+// remember the `@` prefix when referencing aliases
+cy.get('button.createUser').click()
+cy.wait('@createUser')
+// the commands below will not execute until a response is received from `POST /users`
+cy.get('#field.firstName').should('equal', 'Adam')
+
```

Comment:
I'm building on previous examples. Rather than use `@someRoute`, I'm continuing to use `@createUser`, which should reduce **cognitive load**.